Description: Correctly spell the word "associated".
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/doc/reference/libbamf/html/BamfTab.html
+++ b/doc/reference/libbamf/html/BamfTab.html
@@ -355,7 +355,7 @@
 <div class="refsect2">
 <a name="BamfTab--desktop-id"></a><h3>The <code class="literal">“desktop-id”</code> property</h3>
 <pre class="programlisting">  “desktop-id”               <a href="/usr/share/gtk-doc/html/glib/glib-Basic-Types.html#gchar"><span class="type">gchar</span></a> *</pre>
-<p>The Desktop ID assosciated with the application hosted in the remote Tab.</p>
+<p>The Desktop ID associated with the application hosted in the remote Tab.</p>
 <p>Flags: Read</p>
 <p>Default value: NULL</p>
 </div>
--- a/lib/libbamf/bamf-tab.c
+++ b/lib/libbamf/bamf-tab.c
@@ -158,7 +158,7 @@
                               NULL, G_PARAM_READABLE);
   g_object_class_install_property (obj_class, PROP_LOCATION, pspec);
 
-  pspec = g_param_spec_string("desktop-id", "Desktop Name", "The Desktop ID assosciated with the application hosted in the remote Tab",
+  pspec = g_param_spec_string("desktop-id", "Desktop Name", "The Desktop ID associated with the application hosted in the remote Tab",
                               NULL, G_PARAM_READABLE);
   g_object_class_install_property (obj_class, PROP_DESKTOP_ID, pspec);
 
